ppl/src/main/java/org/opensearch/sql/ppl/parser/AstBuilder.java

```diff
@@ -357,10 +363,16 @@ public UnresolvedPlan visitTableSourceClause(TableSourceClauseContext ctx) {
   }
 
   @Override
-  @Generated //To exclude from jacoco..will remove https://github.com/opensearch-project/sql/issues/1019
   public UnresolvedPlan visitTableFunction(OpenSearchPPLParser.TableFunctionContext ctx) {
```

Comment:
Maybe: move to import
```suggestion
  public UnresolvedPlan visitTableFunction(TableFunctionContext ctx) {
```

Reply:
done.